common/src/main/java/uk/nhs/adaptors/common/testutil/FileUtils.java

```diff
@@ -0,0 +1,16 @@
+package uk.nhs.adaptors.common.testutil;
```

Comment:
Should we change the name to `FileUtil` like "singular util" to match the naming convention?